### PR TITLE
[Spotless] Applying Google Code Format for integ-tests #9

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Spotless apply on integ-test (part 2 of 4 integ-test integration).
 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: integ-test/src/test/java/org/opensearch/sql/correctness/tests/OpenSearchConnectionTest.java [hunk 1]
```diff
@@ -63,36 +59,30 @@ public void testCreateTable() throws IOException {
...
     assertEquals("POST", actual.getMethod());
     assertEquals("/test/_bulk?refresh=true", actual.getEndpoint());
     assertEquals(
-        "{\"index\":{}}\n"
-            + "{\"name\":\"John\"}\n"
-            + "{\"index\":{}}\n"
-            + "{\"name\":\"Hank\"}\n",
-        getBody(actual)
-    );
+        "{\"index\":{}}\n" + "{\"name\":\"John\"}\n" + "{\"index\":{}}\n" + "{\"name\":\"Hank\"}\n",
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/correctness/tests/OpenSearchConnectionTest.java:69]:
```suggestion
        "{\"index\":{}}\n{\"name\":\"John\"}\n{\"index\":{}}\n{\"name\":\"Hank\"}\n",
```

File: integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java [hunk 2]
```diff
@@ -61,70 +57,75 @@ protected void init() throws Exception {
...
+                  result.query(String.format("/aggregations/key/buckets/%d/cvalue/value", i)));
+            });
   }
 
   /**
    * todo fix the issue.
    *
-   * @see <a href="https://github.com/opendistro-for-elasticsearch/sql/issues/59">https://github.com/opendistro-for-elasticsearch/sql/issues/59</a>
+   * @see <a
+   *     href="https://github.com/opendistro-for-elasticsearch/sql/issues/59">https://github.com/opendistro-for-elasticsearch/sql/issues/59</a>
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java:83]: a string longer than 100

File: integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java [hunk 3]
```diff
@@ -141,9 +142,9 @@ public static void dumpCoverage() {
   }
 
   /**
-   * As JUnit JavaDoc says:
-   * "The @AfterClass methods declared in superclasses will be run after those of the current class."
-   * So this method is supposed to run before closeClients() in parent class.
+   * As JUnit JavaDoc says: "The @AfterClass methods declared in superclasses will be run after
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java:145]: Add `<br>`

File: integ-test/src/test/java/org/opensearch/sql/ppl/QueryAnalysisIT.java [hunk 4]
```diff
@@ -107,14 +103,12 @@ public void unsupportedAggregationFunctionShouldFailSyntaxCheck() {
...
-   * Commands that fail semantic analysis should throw {@link SemanticCheckException}.
-   */
+  /** Commands that fail semantic analysis should throw {@link SemanticCheckException}. */
   @Test
   public void nonexistentFieldShouldFailSemanticCheck() {
     String query = String.format("search source=%s | fields name", TEST_INDEX_ACCOUNT);
-    queryShouldThrowSemanticException(query, "can't resolve Symbol(namespace=FIELD_NAME, "
-        + "name=name) in type env");
+    queryShouldThrowSemanticException(
+        query, "can't resolve Symbol(namespace=FIELD_NAME, " + "name=name) in type env");
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/ppl/QueryAnalysisIT.java:111]:
```suggestion
        query, "can't resolve Symbol(namespace=FIELD_NAME, name=name) in type env");
```

**Review by Yury-Fridlyand — APPROVED** [review]

File: integ-test/src/test/java/org/opensearch/sql/legacy/QueryIT.java [hunk 5]
```diff
@@ -964,195 +1102,177 @@ public void orderByAscFieldWithSpaceTest() throws IOException {
...
-                "FROM %s " +
-                "WHERE male IS false " +
-                "GROUP BY balance " +
-                "LIMIT 5",
-            TestsConstants.TEST_INDEX_BANK)
-    );
+    JSONObject response =
+        executeQuery(
+            StringUtils.format(
+                "SELECT * " + "FROM %s " + "WHERE male IS false " + "GROUP BY balance " + "LIMIT 5",
```
Comment (GumpacG) [inline integ-test/src/test/java/org/opensearch/sql/legacy/QueryIT.java:1200]:
```suggestion
                "SELECT * FROM %s WHERE male IS false GROUP BY balance LIMIT 5",
```

File: integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java (hunk 2 above)
Comment (MitchellGale) [reply integ-test/src/test/java/org/opensearch/sql/legacy/SQLFunctionsIT.java:83]: It'll be addressed by spotless in final `integ-test` spotless PR if it is a problem.

File: integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java [hunk 6]
```diff
@@ -141,9 +142,9 @@ public static void dumpCoverage() {
...
+   * those of the current class." So this method is supposed to run before closeClients() in parent
```
Comment (MitchellGale) [inline integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java:146]:
```suggestion

```

File: integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java (hunk 3 above)
Comment (MitchellGale) [inline integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java:145]:
```suggestion
   * As JUnit JavaDoc says:<br>
    "The @AfterClass methods declared in superclasses will be run after those of the current class."<br>
    So this method is supposed to run before closeClients() in parent class.
```

**Comment by Yury-Fridlyand** [comment]
Few more comments left

**Comment by MitchellGale** [comment]
> Few more comments left

thanks! Missed those, done now.

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by pjfitzgibbons — APPROVED** [review]

**Pull request merged**